src/main/java/seedu/taassist/logic/commands/BackCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.taassist.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.taassist.model.Model;
+import seedu.taassist.model.moduleclass.ModuleClass;
+
+/**
+ * Exits focus mode.
+ */
+public class BackCommand extends Command {
+
+    public static final String COMMAND_WORD = "back";
+
+    public static final String MESSAGE_ALREADY_EXITED = "Already exited focus mode";
```

Comment:
Perhaps say "Not in focus mode" instead of "Already exited focus mode", since we don't know whether a user has previously visited focus mode or not.